apps/src/code-studio/components/StatusCheckmarkIcon.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import color from '@cdo/apps/util/color';
+import {makeEnum} from '@cdo/apps/utils';
+import PropTypes from 'prop-types';
+
+export const iconStatus = makeEnum('noAction', 'success', 'failure');
```

Comment:
nit: when using enums that have a "nothing" value like this one, it's usually called "none" (e.g., `iconStatus.none`)